### PR TITLE
added blog on event registration

### PR DESCRIPTION
**Pull request by rohitner**
events registration for ROBOTIX 2018

### CODE REVIEW

File: _posts/2017-12-26-registration-process-for-robotix-2018.markdown [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+![](/img/blog/2017/regs-2018/1.png){:.img-responsive}
+
+ * Fill in all the details. **Note:** There are two parts of this. You will need to fill in both the parts.
+
+![](/img/blog/2017/regs-2018/2.png){:.img-responsive}
+
+![](/img/blog/2017/regs-2018/3.png){:.img-responsive}
+
+ * Now you will need to login to your account using the email and passowrd.
```
Comment (shivangag) [inline _posts/2017-12-26-registration-process-for-robotix-2018.markdown]: passwrd -> password

File: _posts/2017-12-26-registration-process-for-robotix-2018.markdown [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  -
+    text: 'E'
+    caption: 'Robotix Events'
+    url: 'https://www.robotix.in/event/'
+    target: '_blank'
+  -
+    text: 'R'
+    caption: 'Register Here'
+    url: 'http://www.ktj.in'
+    target: '_blank'
```
Comment (narayanaditya95) [inline _posts/2017-12-26-registration-process-for-robotix-2018.markdown]: Remove key `target`, rename key `url` to `link`.

File: _posts/2017-12-26-registration-process-for-robotix-2018.markdown [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+layout: post
+slug: registration-for-robotix-2018
+title: 'Registrations for Robotix 2018'
+categories:
+  - 'Robotix 2018'
+actions:
+  -
+    text: 'E'
+    caption: 'Robotix Events'
+    url: 'https://www.robotix.in/event/'
```
Comment (narayanaditya95) [inline _posts/2017-12-26-registration-process-for-robotix-2018.markdown]: `2018.robotix.in/event/`

File: _posts/2017-12-26-registration-process-for-robotix-2018.markdown [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  -
+    text: 'R'
+    caption: 'Register Here'
+    url: 'http://www.ktj.in'
+    target: '_blank'
+---
+
+#### Registration for ROBOTIX 2018 are now open.
+
+For all the event details of Robotix 2018, visit: [Robotix 2018 - Events]({{ site.baseurl }}/event/).
```
Comment (narayanaditya95) [inline _posts/2017-12-26-registration-process-for-robotix-2018.markdown]: `2018.robotix.in/event`

File: _posts/2017-12-26-registration-process-for-robotix-2018.markdown (hunk 3 above)
Comment (narayanaditya95) [reply _posts/2017-12-26-registration-process-for-robotix-2018.markdown]: @shivangag Add a CNAME record on `2018.robotix.in`

**Comment by rohitner** [comment]
Please make sure that `2018.robotix.in` works.

File: _posts/2017-12-26-registration-process-for-robotix-2018.markdown [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+layout: post
+slug: registration-for-robotix-2018
+title: 'Registrations for Robotix 2018'
+categories:
+  - 'Robotix 2018'
+actions:
+  -
+    text: 'E'
+    caption: 'Robotix Events'
+    url: 'https://2018.robotix.in/event/'
```
Comment (narayanaditya95) [inline _posts/2017-12-26-registration-process-for-robotix-2018.markdown]: @shivangag this wouldn't work

Comment (shivangag) [reply _posts/2017-12-26-registration-process-for-robotix-2018.markdown]: Sending a patch.

Comment (shivangag) [reply _posts/2017-12-26-registration-process-for-robotix-2018.markdown]: Done.

**Pull request merged**